api-server/README.md

```diff
@@ -0,0 +1,43 @@
+coepi-backend-aws/api-server
```

Comment:
In the base README, it'd be good to mention that there are two options of running and deploying this server with a link to this readme for API server.